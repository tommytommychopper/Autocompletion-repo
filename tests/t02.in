insert	most	2
insert	best	2
insert	hello	2
insert	special	2
insert	please	2
insert	the	2
insert	better	2
insert	herb	2
insert	speed	2
insert	plagiarism	2
insert	bed	2
insert	thanks	2
insert	modern	2
insert	beautiful	2
insert	spell	2
insert	they	2
insert	bed	2
insert	plural	2
insert	heart	2
insert	spanish	2
insert	spark	2
insert	place	2
insert	beach	2
insert	movie	2
print_inorder
heightPrint
printBF
findPrint	th
findPrint	be
findPrint	mo
findPrint	he
findPrint	sp
findPrint	pl
