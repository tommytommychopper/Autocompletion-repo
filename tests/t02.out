(be){best better bed beautiful bed beach } (he){hello herb heart } (mo){most modern movie } (pl){please plagiarism plural place } (sp){special speed spell spanish spark } (th){the thanks they } 

0 1 0 2 1 0 

0 0 0 0 -1 0 

the thanks they 
best better bed beautiful bed beach 
most modern movie 
hello herb heart 
special speed spell spanish spark 
please plagiarism plural place 
